tests/api_tests/nodes/test_views.py

```diff
@@ -2447,8 +2447,9 @@ def test_return_public_node_pointers_logged_out(self):
         res_json = res.json['data']
         assert_equal(len(res_json), 1)
         assert_equal(res.status_code, 200)
-        assert_in(res_json[0]['attributes']['target_node_id'], self.public_pointer_project._id)
         assert_equal(res.content_type, 'application/vnd.api+json')
+        url = res.json['data'][0]['relationships']['target_node']['links']['related']['href']
+        assert_in('/{}nodes/{}/'.format(API_BASE, self.public_pointer_project._id), url)
```

Comment:
As for `assert_in`, all of these tests using `assert_in` seem fragile, because they would would pass even if the API gave back a different URL like `http://localhost:80/v2/nodes/pgvjq/contributors/`.

What you really want is to ensure the end part of the url (after `localhost:port`) is the same (and that there aren't any other bits). The below suggestion is a hack at 11:30pm; there may be a more elegant option.

```
url_matches = actual_url.endswith(expected_url)
assert_true(url_matches)
```

Since there are a lot of tests that follow this pattern, a lot of tests will need to be updated. (This is where helper methods that DRY common code come in handy!)